pkgs/by-name/pr/protonup-rs/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "protonup-rs";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "auyer";
+    repo = "Protonup-rs";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-wD0R1aVYcLwTg21k3fHu+asTjYq9vI0X9GzyVg5yAvw=";
+  };
+
+  cargoHash = "sha256-6FQDpL74h13eughrB+ZDnWQ3zb2XDRwe/TtOYY4PK9I=";
+
+  doCheck = false;
+
+  meta = {
+    mainProgram = "protonup-rs";
+    description = "A Rust app to Install and Update GE-Proton for Steam, and Wine-GE for Lutris";
+    homepage = "https://github.com/auyer/Protonup-rs";
+    changelog = "https://github.com/auyer/Protonup-rs/releases/tag/v${finalAttrs.version}";
+    license = lib.licenses.gpl3;
+    platforms = [
+      "x86_64-linux"
+      "aarch64-linux"
+    ];
```

If there isn't any particular reason why it wouldn't build for other architectures (e.g. the package contains native binaries), then it's better to allow it to be built on all Linux platforms

```suggestion
    platforms = lib.platforms.linux;
```